rust/processor/src/bq_analytics/gcs_handler.rs

```diff
@@ -13,7 +13,6 @@ use tokio::{
     time::{sleep, timeout, Duration},
 };
 use tracing::{debug, error, info};
-const BUCKET_REGULAR_TRAFFIC: &str = "devnet-airflow-continue";
```

Comment:
i used this to differentiate between routine/continue (get next versions) vs backfill/test. Other way is to use different project

Reply:
let's keep this way? since this would be more straight forward unless using different project gives more benefits.

Reply:
idk for sure the situation but in general it'd be better to rely on metadata about the deployment (e.g. the configuration name, project, etc), just less duplication that way.